core/netparams/defaults.go

```diff
@@ -257,6 +257,13 @@ func defaultNetParams() map[string]value {
 
 		RewardsVestingBaseRate:       NewDecimal(gtD0, lteD1).Mutable(true).MustUpdate("0.01"),
 		RewardVestingMinimumTransfer: NewDecimal(gtD0, lteD1).Mutable(true).MustUpdate("0.01"),
+
+		// Referral program
+		ReferralProgramMaxBenefitTiers:           NewUint(UintGTE(num.NewUint(0)), UintLTE(num.NewUint(100))).Mutable(true).MustUpdate("0"),
+		ReferralProgramMaxReferralRewardFactor:   NewDecimal(gteD0, lteD100).Mutable(true).MustUpdate("0"),
+		ReferralProgramMaxReferralDiscountFactor: NewDecimal(gteD0, lteD100).Mutable(true).MustUpdate("0"),
+		ReferralProgramMaxPartyVolumePerEpoch:    NewUint(UintGTE(num.NewUint(0)), UintLTE(num.NewUint(10000000))).Mutable(true).MustUpdate("0"),
+		ReferralProgramMinStakedVegaTokens:       NewDecimal(gteD1).Mutable(true).MustUpdate("1000000000000000000"),
```

Comment:
can we write in a notation that doesn't have me counting zeros? Something `num.MustDecimalFromString("1e18")`?

Reply:
Done.